move/sources/canvas/token.move

```diff
@@ -95,7 +96,7 @@ module addr::canvas_token {
         config: CanvasConfig,
 
         /// The pixels of the canvas.
-        pixels: SmartTable<u64, Pixel>,
+        pixels: SmartTable<u32, Pixel>,
```

Comment:
FYI these kinds of changes are not allowed when upgrading the contract. So if you change any struct, you'd need to deploy anew.

Reply:
yes ill redeploy